src/System/Taffybar/Information/Network.hs

```diff
@@ -103,7 +103,7 @@ monitorNetworkInterfaces interval onUpdate = void $ do
         let speedInfo = map sampleToSpeeds samples
         onUpdate speedInfo
         return samples
-      doUpdate = MV.modifyMVar_ samplesVar ((>>= doOnUpdate) . updateSamples)
+      doUpdate = MV.modifyMVar_ samplesVar (doOnUpdate <=< updateSamples)
```

Comment:
Is this change necessary? How does this impact anything?

Reply:
Not really, I thought it was clearer, but I will roll it back!.

Reply:
done on rebase!